model/model.go

```diff
@@ -71,6 +71,8 @@ func (model Model) AddDef(sec string, key string, value string) bool {
 		for i := range ast.Tokens {
 			ast.Tokens[i] = key + "_" + strings.TrimSpace(ast.Tokens[i])
 		}
+	} else if sec == "g" {
+		ast.Tokens = strings.Split(ast.Value, ",")
```

Comment:
What's this? If this is fixing a bug, plz raise another PR

Reply:
Now, `ast.Tokens` will be set nil when the section is `g`, so the `len(ast.Tokens)` will always return 0. 
We can not check the length by `len(rule) < len(ast.Tokens)` if not change here.

In another way, We can also add a `tlen`/`tokenLen` field to record the token len when loading the model.

Reply:
got it.